iphone/Classes/TiUIiOSProxy.m

```diff
@@ -715,6 +719,13 @@ -(id)createBlurView:(id)args
 }
 #endif
 
+#ifdef USE_TI_UIIOSSTEPPER
+-(id)createStepper:(id)args
+{
+    return [[[TiUIiOSStepperProxy alloc] _initWithPageContext:[self executionContext] args:args] autorelease];
```

Comment:
I was just making it the same one as all the classes above including the blurView, will change it back to [self pageContext].